global/matlab/Compile_All_MATLAB_code.sh

```diff
@@ -22,6 +22,83 @@
 # See the [LICENSE](https://github.com/Washington-University/HCPpipelines/blob/master/LICENSE.md) file
 #
 
+g_script_name=$(basename "${0}")
+
+# ------------------------------------------------------------------------------
+#  Usage Description Function
+# ------------------------------------------------------------------------------
+
+usage()
+{
+	cat <<EOF
+
+${g_script_name}
+
+  This script can be used to (re)-compile all MATLAB code for functions called by
+  any pipelines that are part of the HCP Pipelines.
+
```

Comment:
Maybe make even more explicit, early in the usage/comments, that use of this script is only necessary if someone wants/needs to RE-compile all the matlab-related scripts.  i.e., that the distribution _already provides_ compiled matlab binaries for use with one specific MCR version, namely R2016b/v91.